packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -95,7 +95,7 @@ function doMpaNavigation(url: string): FetchServerResponseResult {
 
 // TODO: Figure out why this module is included in server page bundles.
```

Comment:
I feel we can move these code into a browser assertion since we're clear about why it's getting imported on server, it's because of layout-router. and remove the TODO comment. The abort controller is also not used in SSR as it's not aborted

```
if (typeof window !== "undefined") { ... }
```